### PR TITLE
Extract index message and copy

### PR DESCRIPTION
**Pull request by nmattia**
This extracts a `showMessage` component/function from the index and extracts the copy too.

The new `showMessage` is also reused in other places that need to display a centered message, and the `data-role` is fixed (in index.ts, the attribute had been copy/pasted but was incorrectly suggesting some authorization was successful).

### CODE REVIEW

File: src/frontend/src/components/message.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { html, TemplateResult } from "lit-html";
+import { ifDefined } from "lit-html/directives/if-defined.js";
```
Comment (peterpeterparker) [inline src/frontend/src/components/message.ts:2]: the explicit `.js` is intended?

File: src/frontend/src/components/message.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const showMessageTemplate = ({
+  message,
+  role,
+}: {
+  message: string;
+  role?: string;
+}): TemplateResult => {
+  return html`<h1
+    style="position: absolute; max-width: 100%; top: 50%; transform: translate(0, -50%);"
```
Comment (peterpeterparker) [inline src/frontend/src/components/message.ts:13]: not for this PR: shouldn't it be a `class`?

File: src/frontend/src/components/message.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { renderPage } from "../utils/lit-html";
+
+const showMessageTemplate = ({
+  message,
+  role,
+}: {
+  message: string;
+  role?: string;
+}): TemplateResult => {
+  return html`<h1
```
Comment (peterpeterparker) [inline src/frontend/src/components/message.ts:12]: not for this PR: should it be a h1 ?

File: src/frontend/src/index.ts [hunk 4]
```diff
@@ -103,17 +106,14 @@ const init = async () => {
...
-        style="position: absolute; max-width: 100%; top: 50%; transform: translate(0, -50%);"
-        data-role="notify-auth-success"
-      >
-        You may close this page.
-      </h1>`,
-      container
-    );
+    showMessage({
+      message: staticCopy.close_page_device_added,
+      role: "notify-device-added",
```
Comment (peterpeterparker) [inline src/frontend/src/index.ts:115]: renamed to `notify-device-added` on purpose right?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

Two questions related to the PR, two just because I noticed these while reading the code.

File: src/frontend/src/components/message.ts (hunk 2 above)
Comment (nmattia) [reply src/frontend/src/components/message.ts:13]: yeah I think that'd make sense. I was just moving code around though but happy to do this in this PR if you prefer

File: src/frontend/src/index.ts (hunk 4 above)
Comment (nmattia) [reply src/frontend/src/index.ts:115]: yep! thanks for paying attention though

File: src/frontend/src/components/message.ts (hunk 3 above)
Comment (nmattia) [reply src/frontend/src/components/message.ts:12]: Good question; probably not. Added to todo list alongside your comment about making the css attributes into a class

File: src/frontend/src/components/message.ts (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/components/message.ts:2]: good to know

**Pull request merged**